### PR TITLE
Ready for Review: Adding essence scores to NPC sheet

### PR DESCRIPTION
**Pull request by phildominguez**
Addresses https://github.com/WookieeMatt/Essence20/issues/234

In this MR:
- Adding essence scores to NPC sheets. They're already part of a template that NPC inherits, so no change to template.json needed.
- General improvements to NPC sheet layout

Testing:
- NPC sheet should have essence scores and function as expected

### CODE REVIEW

File: templates/actor/parts/headers/actor-npc-header.hbs [hunk 1]
```diff
@@ -1,18 +1,9 @@
 {{#>"systems/essence20/templates/actor/parts/headers/actor-header.hbs"}}
 {{#*inline "secondary-fields"}}
-<div class="resource">
+<div class="resource2">
```
Comment (phildominguez) [inline templates/actor/parts/headers/actor-npc-header.hbs:3]: Not seeing `resource2` defined

File: templates/actor/actor-npc-sheet.hbs [hunk 2]
```diff
@@ -12,16 +12,29 @@
...
-        {{!-- Initiative --}}
-        {{> "systems/essence20/templates/actor/parts/actor-initiative.hbs"}}
-
-        {{!-- Movement --}}
-        {{> "systems/essence20/templates/actor/parts/actor-movement.hbs"}}
+      <div class="npc-row">
+        <div class="npc-column-2">
+          {{!-- NPC Essence Scores --}}
+          {{> "systems/essence20/templates/actor/parts/actor-npc-essence-scores.hbs"}}
+          {{> "systems/essence20/templates/actor/parts/actor-npc-defenses.hbs"}}
```
Comment (phildominguez) [inline templates/actor/actor-npc-sheet.hbs:19]: `{{!-- NPC Defenses --}}` above this

File: templates/actor/actor-npc-sheet.hbs [hunk 3]
```diff
@@ -12,16 +12,29 @@
...
+            <div class="resource flex-group-center">
+              <label for="system.health.value" class="resource-label">{{localize 'E20.ActorHealth'}}</label>
+              <div class="resource-content flexrow flex-center flex-between">
+                <input type="text" name="system.health.value" value="{{system.health.value}}" data-dtype="Number" />
+                <span>/</span>
+                <input type="text" name="system.health.max" value="{{system.health.max}}" data-dtype="Number" />
+              </div>
+            </div>
+          </div>
+          {{!-- MOvement --}}
```
Comment (phildominguez) [inline templates/actor/actor-npc-sheet.hbs:33]: `O` -> `o`

File: templates/actor/actor-npc-sheet.hbs [hunk 4]
```diff
@@ -36,6 +49,7 @@
           {{> "systems/essence20/templates/actor/parts/actor-threat-powers.hbs"}}
           {{!-- Perks --}}
           {{> "systems/essence20/templates/actor/parts/actor-perks.hbs"}}
+
```
Comment (phildominguez) [inline templates/actor/actor-npc-sheet.hbs:52]: Remove newline

File: templates/actor/parts/actor-common.hbs [hunk 5]
```diff
@@ -1,6 +1,8 @@
 <div class="flexrow stats-row">
+  <div class="stats-container" style="border-color: {{system.color}}; flex-grow: 0.25;">
   {{!-- Initiative --}}
   {{> "systems/essence20/templates/actor/parts/actor-initiative.hbs"}}
```
Comment (phildominguez) [inline templates/actor/parts/actor-common.hbs:4]: Indent 🙂

**Review by phildominguez — COMMENTED** [review]
@guswadsworth12 Very nice, just minor stuff

File: templates/actor/parts/headers/actor-npc-header.hbs (hunk 1 above)
Comment (guswadsworth12) [reply templates/actor/parts/headers/actor-npc-header.hbs:3]: this was a typo. fixed

File: templates/actor/actor-npc-sheet.hbs (hunk 3 above)
Comment (guswadsworth12) [reply templates/actor/actor-npc-sheet.hbs:33]: what's the issue here?

Comment (guswadsworth12) [reply templates/actor/actor-npc-sheet.hbs]: fixed

**Pull request merged**